### DISCUSSION TITLE
Need to build something in house

### DISCUSSION THREAD

**Discussion by timothywells** [omaha-consulting/winstall#76] [Q&A]
It's pretty simple, I have been wanting to do something EXACTLY like this for myself BUT I've been asked to try and do this as an in-house site on our intranet system. I work for a QA house that works directly with Microsoft. Which means the security for everything is way crazy. Especially when it comes to installing software. I know winstall uses Windows Package Manager and I've reviewed every line of code but your API, so I can say it is relatively safe.

In short, where can I get the API you created so I can replicate it with the specific allowed software information that the QA house I work for is allowed to use?

PS - It's a video game QA company and as such we have access to Xbox dev tools which you can't get "legally" unless you are working with devs or are a dev. Which is one of the major reason it has to be in house.

**Comment by MehediH** [comment]
Hey, winstall API is not available to third parties for free. Currently I am paying for everything out of my own pocket, so need a way to fund the server costs. Feel free to reach out at meh (at) builtbymeh dot com if you want to set something up.
